benchmarks/asv/optimize.py

```diff
@@ -63,7 +63,11 @@ def time_optimize(self, args: str) -> None:
         "inmemory, tpe, 1000",
         "inmemory, cmaes, 1000",
         "sqlite, random, 1000",
-        "cached_sqlite, random, 1000",
```

Comment:
I removed it because `sqlite` and `cached_sqlite` are equivalent in the current implementation.